continuously_train_user_model.py

```diff
@@ -112,11 +114,21 @@ def main(pl_model: LightningModule, user_hash, config: configs.BaseConfig) -> Li
 
     online_sessions = get_stored_sessions(config)
 
+    session_metrics_list = []
     for session_idx, current_trial_episodes in enumerate(online_sessions):
-        process_session(config, current_trial_episodes, logger, pl_model, session_idx)
+        episode_metrics = process_session(config, current_trial_episodes, logger, pl_model, session_idx)
+        session_metrics_list.append(episode_metrics)
 
-    wandb.run.log({}, commit=True)
-    return pl_model
+    mean_metrics = {}
+    if len(session_metrics_list) > 1:
+        for key in session_metrics_list[0].keys():
+            mean_metrics[f"continuous/{key}"] = np.mean([metric[key] for metric in session_metrics_list])
+
+    print(mean_metrics)
+    wandb.run.log(mean_metrics, commit=True)
+    score = mean_metrics['continuous/validation/f1']
```

Comment:
wandb only folds one level, either first or last, this line is correct but fyi

Reply:
can't we just give a metrics prefix to the pl.Trainer? Then all metrics are logged for example with "user_hast/finetune/f1" and are thus unique. Also, we wouldn't need this extra logging step here, it would automatically happen in validate

Reply:
solved in chat, right?